terra-workspace-manager/sa.tf

```diff
@@ -49,13 +55,25 @@ resource "google_project_iam_member" "app" {
   role     = local.app_sa_roles[count.index]
   member   = "serviceAccount:${google_service_account.app[0].email}"
 }
+// TODO(PF-156): Remove this once WM uses RBS
 resource "google_folder_iam_member" "app" {
   count    = local.create_folder ? length(local.app_folder_roles) : 0
   provider = google.target
   folder   = google_folder.workspace_project_folder[0].id
   role     = local.app_folder_roles[count.index]
   member   = "serviceAccount:${google_service_account.app[0].email}"
 }
+# Grant WorkspaceManager Service App Service Account permission to modify resource in folder.
+resource "google_folder_iam_member" "app_folder_roles" {
+  // Skip if google_folder variable is not present.
```

Comment:
what google_folder variable?

Reply:
Not sure... removing.

Reply:
Not sure... must have been a lingering copy or something...